hibernate-core/src/main/java/org/hibernate/metamodel/mapping/internal/EmbeddedAttributeMapping.java

```diff
@@ -406,6 +406,7 @@ private static PropertyAccess getPropertyAccess(
 					true );
 		}
 		else {
+//			throw new NotYetImplementedFor6Exception();
```

Comment:
Unnecessary

Reply:
yeap an oversight :) I'm going to delete it thanks